.github/workflows/status_embed.yaml

```diff
@@ -0,0 +1,63 @@
+name: Status Embed
+
+on:
+  workflow_call:
+    inputs:
+      job_status:
+        required: true
+        type: string
+    secrets:
+      webhook_token:
+        required: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  status_embed:
+    # We need to send a status embed whenever the workflow
+    # sequence we're running terminates. There are a number
+    # of situations in which that happens:
+    #
+    # 1. We reach the end of the Deploy workflow, without
+    #    it being skipped.
+    #
+    # 2. A `pull_request` triggered a Lint & Test workflow,
+    #    as the sequence always terminates with one run.
+    #
+    # 3. If any workflow ends in failure or was cancelled.
+    if: >-
+      (github.workflow == 'Deploy' && ${{ inputs.job_status }} != 'skipped') ||
+      github.event_name == 'pull_request' ||
+      ${{ inputs.job_status }} == 'failure' ||
+      ${{ inputs.job_status }} == 'cancelled'
+    name:  Send Status Embed to Discord
+    runs-on: ubuntu-latest
+
+    steps:
+      # Send an informational status embed to Discord instead of the
+      # standard embeds that Discord sends. This embed will contain
+      # more information and we can fine tune when we actually want
+      # to send an embed.
+      - name: GitHub Actions Status Embed for Discord
+        uses: SebastiaanZ/github-status-embed-for-discord@v0.2.1
+        with:
+          # Our GitHub Actions webhook
+          webhook_id: '784184528997842985'
```

Comment:
This should be an input, so the caller can determine what channel it gets sent to

Reply:
I think setting it as a variable is better, that way we'd avoid having the `with` statements scattered all over the over workflows that'll call this one.

At worst, we can have it as an input, and default to the variable. But shouldn't be needed in our case